.env.example

```diff
@@ -112,7 +112,7 @@ NEXT_PUBLIC_POSTHOG_HOST=
 PLAIN_API_KEY=
 PLAIN_API_URL=https://api.plain.com/v1
 PLAIN_HMAC_SECRET_KEY=
-PLAIN_CHAT_ID=
+NEXT_PUBLIC_PLAIN_CHAT_ID=
```

Comment:
Since we use this client side, it is safe to make it public. This way we can know before hand if Plain is setup or not.